detections/network/windows_remote_desktop_network_bruteforce_attempt.yml

```
@@ -0,0 +1,61 @@
+name: Windows Remote Desktop Network Bruteforce Attempt
+id: 908bf0d5-0983-4afd-b6a4-e9eb5d361a7d
+version: 6  
```

Since this a replacement detection, it should start as version 1

```suggestion
version: 1
```